bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/view/obs/ObsListModel.scala

```diff
@@ -181,15 +181,18 @@ case class ObsListModel(all:List[Observation], band:Band, gs:(ObsListGrouping[_]
 
   }
 
+  def cut(elem: ObsListElem): ObsListModel = elem match {
+    case target: ObsGroup[_] =>
+      val (_, toClear) = Seq(gs._1, gs._2, gs._3).splitAt(depth(target))
 
+      def clear(o: Observation) =
+        toClear.foldLeft(o.copy(progTime = None))((o, g) => g.clear(o))
 
-  def cut(elem:ObsListElem):ObsListModel = elem match {
-    case target:ObsGroup[_] =>
-      val (_, toClear) = Seq(gs._1, gs._2, gs._3).splitAt(depth(target))
-      def clear(o:Observation) = (o.copy(progTime = None) /: toClear)((o, g) => g.clear(o))
       val os = selected(target)
-      (this /: os.zip(os.map(clear)))(_.replacePair(_))
-    case o:ObsElem          => this - o
+      os.zip(os.map(clear)).foldLeft(this)(_.replacePair(_))
+    case o:ObsElem          =>
```

Comment:
I loathe `:/` and `/:`, and I believe current Scala style guides recommend against using them.